[BUG] Snapshot build is broken

**Describe the bug**

Coming from https://github.com/opensearch-project/opensearch-build/pull/189, attempting to build a snapshot version of this plugin:

```
./gradlew assemble --no-daemon --refresh-dependencies -DskipTests=true -Dopensearch.version=1.0.0-SNAPSHOT -Dbuild.snapshot=true
```

Fails to find the non-snapshot depenency. It should be looking for the snapshot dependency of OpenSearch.

```
To honour the JVM settings for this build a new JVM will be forked. Please consider using the daemon: https://docs.gradle.org/6.6.1/userguide/gradle_daemon.html.
Daemon will be stopped at the end of the build stopping after processing

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'opensearch-sql'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not find org.opensearch.gradle:build-tools:1.0.0.
     Searched in the following locations:
       - file:/home/ubuntu/.m2/repository/org/opensearch/gradle/build-tools/1.0.0/build-tools-1.0.0.pom
       - https://repo.maven.apache.org/maven2/org/opensearch/gradle/build-tools/1.0.0/build-tools-1.0.0.pom
       - https://jcenter.bintray.com/org/opensearch/gradle/build-tools/1.0.0/build-tools-1.0.0.pom
       - https://plugins.gradle.org/m2/org/opensearch/gradle/build-tools/1.0.0/build-tools-1.0.0.pom
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 6s
Traceback (most recent call last):
  File "./bundle-workflow/python/build.py", line 56, in <module>
    builder.build(manifest.build.version, arch, args.snapshot)
  File "/home/ubuntu/source/opensearch-project/opensearch-build/dblock-opensearch-build/bundle-workflow/python/build_workflow/builder.py", line 39, in build
    self.git_repo.execute(build_command)
  File "/home/ubuntu/source/opensearch-project/opensearch-build/dblock-opensearch-build/bundle-workflow/python/build_workflow/git_repository.py", line 30, in execute
    subprocess.check_call(command, cwd = self.dir.name, shell = True)
  File "/usr/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '/home/ubuntu/source/opensearch-project/opensearch-build/dblock-opensearch-build/bundle-workflow/python/../scripts/bundle-build/standard-gradle-build/build.sh -v 1.0.0 -a x64 -s true -o artifacts' returned non-zero exit status 1.
```

**Expected behavior**
Build to pickup snapshot version of OpenSearch.

--- a/build.gradle
+++ b/build.gradle
@@ -26,7 +26,7 @@
 
 buildscript {
     ext {
-        opensearch_version = "1.0.0"
+        opensearch_version = System.getProperty("opensearch.version", "1.0.0")
     }
 
     repositories {
@@ -55,12 +55,16 @@ repositories {
 }
 
 ext {
-    opensearchVersion = '1.0.0'
+    opensearchVersion = System.getProperty("opensearch.version", "1.0.0")
     isSnapshot = "true" == System.getProperty("build.snapshot", "true")
 }
 
 allprojects {
-    version = "${opensearchVersion}.0"
+    version = "${opensearchVersion}" - "-SNAPSHOT" + ".0"
+
+    if (isSnapshot) {
+        version += "-SNAPSHOT"
+    }
 
     plugins.withId('java') {
         sourceCompatibility = targetCompatibility = "1.8"

